lib/file-perms.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary
```

Comment:
Balabit it and balabit sarl should also be converted to Balabit, without entity form or location.